invenio_rdm_records/templates/semantic-ui/invenio_notifications/grant-user-access.create.jinja

```diff
@@ -0,0 +1,68 @@
+{% set record = notification.context.record %}
+{% set receiver = notification.context.receiver %}
+{% set message = notification.context.message | safe if notification.context.message else '' %}
+{% set permission = notification.context.permission %}
+
+{% set record_title = record.metadata.title %}
+{% set record_link = record.links.self_html %}
+{% set account_settings_link = "{ui}/account/settings/notifications".format(
+    ui=config.SITE_UI_URL
+    )
+%}
+
+{%- block subject -%}
```

Comment:
can you give a screenshot of how the email looks like when rendered please?

Reply:
Provided in the PR description 😃